### DISCUSSION TITLE
cant seem to get a matcher to work

### DISCUSSION THREAD

**Discussion by phishy** [zendesk/laika#2] [Q&A] [answered]
I tried matching with the everything interceptor:

```
      const absolutelyEverythingInterceptor = apolloTestingToolkit.intercept();
      absolutelyEverythingInterceptor.mockResult({
        result: {
          data: {
            users: [
              { id: 1, name: 'Mouse' },
              { id: 2, name: 'Bamboo' }
            ]
          }
        }
      });
```

and received:

`your-client-name: response:query        1.7s passthrough FinishSignup (undefined) `

Ideas?

**Comment by niieani** [comment, marked as answer]
Hi @phishy!

Not sure why it's not working for you, here's an example [CodeSandbox](https://codesandbox.io/s/laika-example-3yyq0?file=/src/index.tsx) that also has an everything interceptor which works correctly.

If you could give your full example/reproduction in a CodeSandbox format, we can try to figure out why it's not working.
